integrations/object_store/README.md

```diff
@@ -12,7 +12,7 @@ first you need to add the following dependencies to your `Cargo.toml`:
 [dependencies]
 object_store = "0.9.0"
 object_store_opendal = "0.43.1"
-opendal = { version = "0.46.0", features = ["services-s3"] }
+opendal = { version = "0.47.0", features = ["services-s3"] }
```

Comment:
The whole example is outdated.

Reply:
This should be a separated issue to fix ... Let me file an issue and see if we can fix it before the release.

Reply:
https://github.com/apache/opendal/issues/4703

Reply:
Push https://github.com/apache/opendal/pull/4701/commits/48c6fa92e2dc07dcbd07e460949fdc118a616966 for this.